README.md

```diff
@@ -73,3 +79,67 @@ Note: a `.tool-versions` file exists which contains the current supported ruby v
 * `asdf plugin-add ruby`
 * `asdf install`
 
+## Running the app
+
+```
+rails s
```

Comment:
Do you really use this to run the app or is the preferred method in development now `bin/dev`?